src/components/Commits/CommitsListView.tsx

```diff
@@ -72,13 +105,15 @@ const CommitsListView: React.FC<CommitsListViewProps> = ({
 
   return (
     <PageSection padding={{ default: 'noPadding' }} variant={PageSectionVariants.light} isFilled>
-      <Title size="lg" headingLevel="h3" className="pf-c-title pf-u-mt-lg pf-u-mb-sm">
-        {recentOnly ? 'Recent commits' : 'Commits'}
-      </Title>
+      {recentOnly ? (
+        <Title size="lg" headingLevel="h3" className="pf-c-title pf-u-mt-lg pf-u-mb-sm">
+          {recentOnly ? 'Recent commits' : 'Commits'}
+        </Title>
+      ) : null}
       <Text className="pf-u-mb-lg">
-        Monitor your commits and their pipeline progression across all components.{' '}
+        Events progression triggered by the commit.{' '}
         <Button variant="link" className="pf-u-pl-sm">
-          Learn more
+          Read more
         </Button>
```

Comment:
<img width="418" alt="image" src="https://user-images.githubusercontent.com/9964343/211744880-ff578f06-f1cb-4957-a48d-a0475bc9e74f.png">

Clicking on the `Read more` link does nothing but I think should open a getting started with commit modal like we have in commit details page, @MariaLeonova Please confirm?


<img width="1279" alt="image" src="https://user-images.githubusercontent.com/9964343/211744800-ac522eb6-88cb-40de-820a-c49a00dc591f.png">